build/clean_build.sh

```diff
@@ -63,6 +63,7 @@ fi
 cd "$tc_dir"
 if [ -d "${tc_volume}/.git" ]; then
 	# In case the mirrored repo already exists, remove gitignored files
+	git config --global --add safe.directory '*'
```

Comment:
The https://github.com/apache/trafficcontrol/blob/191c0faeca8f5c48d42deb566970a69a13ea73dd/build/clean_build.sh#L65 comment applies to https://github.com/apache/trafficcontrol/blob/191c0faeca8f5c48d42deb566970a69a13ea73dd/build/clean_build.sh#L67 so https://github.com/apache/trafficcontrol/blob/191c0faeca8f5c48d42deb566970a69a13ea73dd/build/clean_build.sh#L66 should be moved above the comment.